index.js

```diff
@@ -72,6 +72,14 @@ function nestRules ({ rules, target, nested = [], mediaQuery = false, mixinsOnly
         else ruleset = [[`@at-root ${prefix}#{&}${suffix.replace('&', '#{&}')}`, rule]]
       } else if (suffix) {
         ruleset = [[`&${suffix}`, rule]]
+      } else {
+        // https://sass-lang.com/documentation/breaking-changes/mixed-decls/
+        if (target === 'scss') {
+          const isNestedSelector = parent[1].some(([selector]) => selector.startsWith('&'))
```

Comment:
Burde man sneket inn en sjekk eller to på referanse til `parent[1].` så den kanskje er litt mindre eksplosiv?

Reply:
Den blir brukt som en tuple, så jeg tror den skal være safe. Men det skader ikke å være litt forsiktig, i mangel av typer som garanterer det.